Agoda.Frameworks.DB/StoredProc.cs

```diff
@@ -49,4 +49,19 @@ public virtual SpParameter[] GetParameters(TRequest parameters)
                 .ToArray();
         }
     }
+
+    public class IAmNotAStoredProc : IStoredProc
+    {
+        public IAmNotAStoredProc(string dbName, string storedProcedureName, int commandTimeoutSecs, int maxAttemptCount)
+        {
+            DbName = dbName;
+            StoredProcedureName = storedProcedureName;
```

Comment:
Why does it have a `storedProcedureName` if it's not a stored procedure? :)

Reply:
ist the way it was designed (to only support SP), hence my tough in cheek naming of the class above